src/main/java/com/wsh/mogak/global/auth/AuthDetails.java

```diff
@@ -0,0 +1,48 @@
+package com.wsh.mogak.global.auth;
+
+import com.wsh.mogak.domain.member.entity.Member;
+import lombok.AllArgsConstructor;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+
+@AllArgsConstructor
+public class AuthDetails implements UserDetails {
+    private Member member;
+
+    @Override
+    public Collection<? extends GrantedAuthority> getAuthorities() {
+        return null;
+    }
+
+    @Override
+    public String getPassword() {
+        return null;
+    }
+
+    @Override
+    public String getUsername() {
+        return member.getName();
```

Comment:
```suggestion
        return member.getEmail();
```

principle은 email인거 같아요

Reply:
[1f8e374](https://github.com/TeamWSH-Moida/Mogak-Backend/pull/5/commits/1f8e3743059f640d4d289c39c14062dcea615ca3)
수정하였습니다!